home/management/commands/change_related_tag_to_related_page.py

```diff
@@ -22,15 +23,23 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
-        pages = ContentPage.objects.live().filter(
-            tags__name__startswith=self.TAG_PREFIX
+        pages = (
+            ContentPage.objects.live()
+            .filter(tags__name__startswith=self.TAG_PREFIX)
+            .distinct()
         )
         for page in pages:
-            related_pages = set(p.value.id for p in page.related_pages)
+            related_pages = set(p.value and p.value.id for p in page.related_pages)
```

Comment:
Isn't this going to be a set of boolean values?

Reply:
If `p.value` is `None`, then it's `None`, but if `p.value` is truthy, then it's `p.value.id`, which is an integer. Basically `p.value.id if p.value else p.value`. Not sure if an `if` is a bit clearer here?

Reply:
ok, thanks. I do think an `if` would be clearer